Godeps/_workspace/src/github.com/onsi/ginkgo/ginkgo/bootstrap_command.go

```diff
@@ -11,7 +11,7 @@ import (
 
 	"go/build"
 
-	"github.com/onsi/ginkgo/ginkgo/nodot"
+	"github.com/ably/ably-go/Godeps/_workspace/src/github.com/onsi/ginkgo/ginkgo/nodot"
```

Comment:
> it would be quite a pain when running godep update my_package/path to have to update this line again. And we would have to make sure that any new import is actually going to the Godeps folder.

`godep` knows how to rewrite the paths and add code to the `Godeps` directory (I used `godep` to make this change).